oraclesrv/views.py

```diff
@@ -6,13 +6,27 @@
 from flask_discoverer import advertise
 
 import json
+import time
 
-from oraclesrv.utils import get_solr_data_recommend, get_solr_data_match, get_solr_data_match_doi, get_solr_data_match_thesis
-from oraclesrv.score import clean_data, score_match, encode_author, format_author, score_match_doi
+from adsmsg import DocMatchRecordList
+from google.protobuf.json_format import Parse, ParseError
+
+from oraclesrv.utils import get_solr_data_recommend, add_records, del_records
+from oraclesrv.keras_model import create_keras_model, load_keras_model
+from oraclesrv.doc_matching import DocMatching, get_requests_params
 
 
 bp = Blueprint('oracle_service', __name__)
 
+# @bp.before_app_first_request
```

Comment:
`# @bp.before_app_first_request` confused me; better to remove that comment

Reply:
done.